account/tests/urls_tests/test_profile.py

```diff
@@ -0,0 +1,13 @@
+import pytest
+
+
+FAKE_ID = 10000000
```

Comment:
I see that you use FAKE_ID only in test_professional.py and not in this file..
Consider moving it there..